console/src/views/Dashboard.js

```diff
@@ -76,7 +76,7 @@ const DashboardBase = kind({
 										Tire psi
 									</Item>
 								</Cell>
-								<Cell component={Divider} shrink startSection />
+								<Cell component={Heading} shrink startSection />
```

Comment:
The `startSection` is not a valid property for `Heading`.  It should be removed or replaced with something else.  Not sure which is appropriate.  There are several appearances of this below.

Reply:
The `startSection` is replaced by `showLine` property in `Heading`